src/app/portlets/content-types/layout/content-types-layout.component.spec.ts

```diff
@@ -31,40 +33,70 @@ class TestDotIframe {
     @Input() src: string;
 }
 
+@Component({
+    selector: '<test></test>',
```

Comment:
Should this be just 'test'?

Reply:
really no matter what selector you use, but maybe be should define a standard